### ISSUE TITLE
Installation failure in v2.1.2

### ISSUE THREAD

**Issue by ddolcimascolo** [JCMais/node-libcurl#226]
Hi guys,

We just upgraded to v2.1.2 a few hours ago and our build started to fail

Relevant log lines:
```
$ npm i
 > node-libcurl@2.1.2 install /builds/stordata/collectors/node_modules/node-libcurl
 > node-pre-gyp install --fallback-to-build
 node-pre-gyp WARN Using request for node-pre-gyp https download 
 [node-libcurl] Success: "/builds/stordata/collectors/node_modules/node-libcurl/lib/binding/node_libcurl.node" is installed via remote
 > node-libcurl@2.1.2 postinstall /builds/stordata/collectors/node_modules/node-libcurl
 > node scripts/postinstall
 node-libcurl WARN Could not finish postinstall 
 node-libcurl ERR! Error: Command failed: node "/builds/stordata/collectors/node_modules/node-libcurl/dist/index.js"
 internal/modules/cjs/loader.js:628
     throw err;
     ^
 Error: Cannot find module './enum/CurlCode'
 Require stack:
 - /builds/stordata/collectors/node_modules/node-libcurl/dist/Curl.js
 - /builds/stordata/collectors/node_modules/node-libcurl/dist/index.js
     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15)
     at Function.Module._load (internal/modules/cjs/loader.js:527:27)
     at Module.require (internal/modules/cjs/loader.js:683:19)
     at require (internal/modules/cjs/helpers.js:16:16)
     at Object.<anonymous> (/builds/stordata/collectors/node_modules/node-libcurl/dist/Curl.js:19:20)
     at Module._compile (internal/modules/cjs/loader.js:777:30)
     at Object.Module._extensions..js (internal/modules/cjs/loader.js:788:10)
     at Module.load (internal/modules/cjs/loader.js:643:32)
     at Function.Module._load (internal/modules/cjs/loader.js:556:12)
     at Module.require (internal/modules/cjs/loader.js:683:19) {
   code: 'MODULE_NOT_FOUND',
   requireStack: [
     '/builds/stordata/collectors/node_modules/node-libcurl/dist/Curl.js',
     '/builds/stordata/collectors/node_modules/node-libcurl/dist/index.js'
   ]
 }
```

Can you advise on what went wrong?

Thanks,
David

**Comment by JCMais** [comment]
Hi @ddolcimascolo , thanks for opening this issue.

This was in fact an issue with the release, due to caching during the build process, some files were not generated correctly, I've just released `2.1.3` with this fixed, it should take a few minutes until all prebuilt binaries are available.

`2.1.2` is deprecated, as it's broken.

**Comment by ddolcimascolo** [comment]
Thx for the quick response @JCMais !

Regards,
David
